clients/client-python/gravitino/api/credential/azure_account_key_credential.py

```diff
@@ -0,0 +1,88 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from abc import ABC
+from typing import Dict
+
+from gravitino.api.credential.credential import Credential
+from gravitino.utils.precondition import Precondition
+
+
+class AzureAccountKeyCredential(Credential, ABC):
+    """Represents Azure account key credential."""
+
+    AZURE_ACCOUNT_KEY_CREDENTIAL_TYPE: str = "azure-account-key"
+    _GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME: str = "azure-storage-account-name"
+    _GRAVITINO_AZURE_STORAGE_ACCOUNT_KEY: str = "azure-storage-account-key"
+
+    def __init__(self, credential_info: Dict[str, str], expire_time_in_ms: int):
+        self._account_name = credential_info[self._GRAVITINO_AZURE_STORAGE_ACCOUNT_NAME]
+        self._account_key = credential_info[self._GRAVITINO_AZURE_STORAGE_ACCOUNT_KEY]
+        Precondition.check_string_not_empty(
+            self._account_name, "The Azure account name should not be empty"
+        )
+        Precondition.check_string_not_empty(
+            self._account_key, "The Azure account key should not be empty"
+        )
+        Precondition.check_argument(
+            expire_time_in_ms == 0,
```

Comment:
This means that the `expire_time_in_ms` has to be provided and it has to be `0`.
Why? Can we simply remove it from the parameters?

Reply:
The credential is generated by the Gravitino server and the expire time is expected to be zero for azure account key credential, so I think it's necessary to keep it.